src/platform/nxp/k32w/k32w0/KeyValueStoreManagerImpl.cpp

```diff
@@ -38,7 +38,7 @@ namespace DeviceLayer {
 namespace PersistedStorage {
 
 /* TODO: adjust these values */
-constexpr size_t kMaxNumberOfKeys  = 30;
+constexpr size_t kMaxNumberOfKeys  = 60;
```

Comment:
This may fail for products that use more non-volatile space, and more group configurations.

Reply:
Updated the maximum number of keys to a large value (125) such that we expect to accommodate most use-cases. No additional memory is allocated as the memory will be dynamically allocated depending on the actual required number of keys.